pkg/domain/infosync/info.go

```diff
@@ -200,7 +200,9 @@ func GlobalInfoSyncerInit(
 	skipRegisterToDashBoard bool,
 ) (*InfoSyncer, error) {
 	if pdHTTPCli != nil {
-		pdHTTPCli = pdHTTPCli.WithRespHandler(pdResponseHandler)
+		pdHTTPCli = pdHTTPCli.
+			WithCallerID("tidb-info-syncer").
```

Comment:
Do we need to add a constant to this `CallerID`?

Reply:
It seems that currently only this place will use this string. I feel like const is not that necessary?